docs/installation.rst

```diff
@@ -0,0 +1,386 @@
+       :alt: Launching Windows PowerShell from the menu
+
+Once you have launched the Terminal, the window that opens will show the command prompt, where we will input all the commands form now on. The following are a couple of examples of what it looks like, typically, but it might be a bit different in your system depending on how it is configured:
+
+- Linux and MacOS Terminal:
+
+.. code-block:: bash
+
+    your_user_name@computer_name:~$
+
+- Windows PowerShell:
+
+.. code-block:: powershell
+
+    PS C:\Users\your_user_name>
+
+.. note::
+
+    For simplicity, we will be excluding the command prompt in the following sections whenever we indicate that a command should be executed in the terminal.
+
+Installing a compatible Python version
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MUSE needs Python to run but, for now, it only works with versions 3.8 and 3.9, so the next step is to install a suitable version of Python.
+
+.. note::
+
+    Windows users should disable the alias for Python that comes by default with Windows and that will try to install Python from the Microsoft Store everytime we write ``python`` in the terminal. To do so, press the ``Windows key`` and start typing ``alias``, when it shows up, click in ``Manage app execution aliases``. In the window that opens, disable the two entries related to Python, like in the image.
+
+    .. image:: figures/disable_python_alias.png
+        :width: 400
+        :align: center
+        :alt: Screen to disable the Python aliases defined by Windows.
+
+The first thing will be to check if you already have a suitable python version installed. Open a terminal and run:
+
+.. code-block:: bash
+
+    python --version
+
+If the output is ``Python 3.8.X`` or ``Python 3.9.X``, where ``X`` is any number, then you have a version of Python compatible with MUSE and you can skip this section altogether. Move to `Installing pipx`_. In any other case, keep reading.
+
+There are multiple ways of installing Python, as well as multiple distributions. Here we have opted for the one that we believe is simplest, requires the smallest downloads and gives the maximum flexibility: using ``pyenv``.
+
+.. note::
+
+    If you have Anaconda Python installed, then you can use it instead of ``pyenv`` to create an environment with a suitable Python version. Go to section `Creating a conda virtual environment`_ and jump to `Installing pipx`_ when it is completed.
+
+Installing ``pyenv``
+^^^^^^^^^^^^^^^^^^^^
+
+`pyenv <https://github.com/pyenv/pyenv>`_ (`pyenv-win <https://pyenv-win.github.io/pyenv-win/>`_ for Windows) is a tool that lets you install and manage different python versions. It is small, unobtrusive and self-contained, and it is available for the three operative systems.
+
+To install it, follow these steps:
+
+- **Linux**: In this case, you will need to clone the GitHub repository using ``git``. Most Linux distributions come with ``git`` installed, so this should work out of the box:
+
```

Comment:
This requires that git is installed

Reply:
Yes, which is available by default in most Linux distributions, as explained in the text.